app/handlers/testConfig.py

```diff
@@ -30,39 +30,37 @@ class TestConfigHandler(BaseHandler):
 
```

Comment:
I have made similar changes, might be a merge conflict in this one. Looks good though 👍